app/assets/stylesheets/application.scss

```diff
@@ -655,11 +655,6 @@ span.inactive {
 	}
 }
 
-.password-input {
-	height: 44px;
-	width: 480px;
-}
-
```

Comment:
This looks like it was added for a specific reason. How did you ensure that this didn't cause any regression?

Reply:
The `show` button was placed on top of the password input field by setting a particular size for the field. In mobile view, because of the above set width, the password field goes beyond the screen as well as the `show` button. 
After removing these, the size of the field became relative and adjust with the change in dimension of the window.
Fixed the position of the `show` button by specifying `right-margin` instead of `left-margin`

Reply:
Cool.